html/changelogs/wickedcybs_tajhair.yml

```diff
@@ -0,0 +1,8 @@
+author: WickedCybs
+
+delete-after: True
+
+changes:
+  - rscadd: "Added some new Taj hairstyles and variations. Mainly variations."
+  - rscadd: "A second preview that can be viewed in all different directions has been added to the tajara dmi. This is only really of interest to spriters."
+  - tweak: "A lot of the current taj alt hairs had their names changed to make it more clear what is similar in style"
```

Comment:
Since people will need to reselect the styles, this should be mentioned in the changelog